code/modules/reagents/reagent_dispenser.dm

```diff
@@ -160,10 +160,6 @@
 	tank_volume = 500
 	var/paper_cups = 25 //Paper cups left from the cooler
 
-/obj/structure/reagent_dispensers/water_cooler/New()
-	..()
-	name = "[reagent_id] cooler"

```

Comment:
's better than ```reagent_id```, which it should never have been given that's the non-ic reference to it.